examples/github-oauth/README.md

```diff
@@ -0,0 +1,15 @@
+# GitHub OAuth Application Example
+
+This example assumes you already have EveryAuth configured in your development environment. In case you don’t, follow the [configuration steps](https://github.com/fusebit/everyauth-express#getting-started).
+
+Install dependencies
+
+```shell
+npm i
+```
+
+Run the example
+
+```shell
+node .
+```
```

Comment:
Added

Reply:
Let's add some screenshots here as well so that people can look at the README and see what's going to happen w/o having to actually run the code themselves.

Reply:
Done!